app/controllers/application_controller.rb

```diff
@@ -4,16 +4,11 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   def default_url_options
-    options = {}
-    if ENV['APP_HOST'].present?
-      options.merge!(host: ENV['APP_HOST'])
+    env_prefix = 'APP_URL_OPTIONS_'
+    ENV.select { |k, v| (k.start_with? env_prefix) && (not v.empty?) }.
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'k'
* [UncommunicativeVariableName] has the variable name 'v'